enricher/standard/src/main/java/io/fabric8/maven/enricher/standard/DefaultServiceEnricher.java

```diff
@@ -141,14 +135,34 @@ private void addServices(KubernetesListBuilder builder) {
         }
     }
 
+    private String getAppName(){
+        try {
+            if (getContext().getProjectClassLoaders().isClassInCompileClasspath(true)) {
+                Properties properties = SpringBootUtil.getSpringBootApplicationProperties(getContext().getProjectClassLoaders().getCompileClassLoader());
+                return properties.getProperty("spring.application.name");
+            }
+        } catch (Exception ex) {
```

Comment:
Can you narrow down to a more specific Exception Class type ? Also, Just `e` suffices as the variable name here.